CONTRIBUTING.rst

```diff
@@ -87,6 +87,10 @@ This section provides general guidance for developing code for the project. The
 
 - Use meaningful names for variables, functions and classes.
 
+- Write docstring for the new functions you ma
+    - We use Google Style Docstring, see [`Tutorial <https://google.github.io/styleguide/pyguide.html>`_] [`Example <https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html>`_].
+    - We would also add doctests for each non-private function. For details, you may refer to [`here <https://docs.python.org/3/library/doctest.html>`_].
```

Comment:
`We would also add doctests ` -> `We implement doctests`, otherwise seems optional